src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,42 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include <string>
 
 
 using namespace std;
 
 int t07_ip() {
+	string s;
+	string number;
+	int now = 0;
+	int next = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'next' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4721051)